packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -453,6 +769,15 @@ const DialogueCheckBox: React.FC<{
     if (response.data.upsertUserDialogueCheck.match) {
       void handleNewMatchAnonymisedAnalytics()
       setShowConfetti(true);
+      openDialog({
+        componentName: 'NextStepsDialog',
+        componentProps: {
+          userId: currentUser?._id,
+          targetUserId,
+          targetUserDisplayName,
+          dialogueCheckId: checkId!
```

Comment:
Better to avoid asserting not null here (and everywhere) and find another way to make it safe. Likely checkId is in the response.

Reply:
@b0b3rt reopening, I'm not sure if this one is supposed to be blocking or not

Reply:
Not sure why it was closed prematurely. All instances of null assertion have been fixed now (you can cmd+f for checkId! in the file). Closing for real! :)